console/controllers/MigrationController.php

```diff
@@ -325,7 +335,7 @@ public function actionData(array $tables)
 
                         $this->prepareFile(['columns' => $pcolumns, 'rows' => $prows]);
 
-                        return self::EXIT_CODE_NORMAL;
+                        //return self::EXIT_CODE_NORMAL;
```

Comment:
every callable action must return an exit code normal or error.

Reply:
this stop on first iteration foreach().